backend/src/models/organization.ts

```diff
@@ -64,6 +61,21 @@ export class Organization extends BaseEntity {
   })
   granularScans: Scan[];
 
+  @ManyToMany((type) => OrganizationTag, (tag) => tag.organizations, {
+    onDelete: 'CASCADE',
+    onUpdate: 'CASCADE'
+  })
+  tags: OrganizationTag[];
+
+  @ManyToOne((type) => Organization, { onDelete: 'CASCADE', nullable: true })
+  parentOrganization: Organization;
```

Comment:
I think we should specify in a comment here that organizations with `parentOrganization` set to null are labeled "Organizations" in the UI, but organizations with a non-null value of `parentOrganization` are called "Teams" in the UI.